Pully Penelope/Assets/Scripts/GuillotineSoundPlayer.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GuillotineSoundPlayer : MonoBehaviour
+{
+    [Tooltip("The sound that the guillotine makes.")]
```

Comment:
love that you're using tooltips but you don't have to put them on things that are self explanitory/already well named